packages/client/integration/ProtectionRecovery.spec.ts

```diff
@@ -51,43 +68,95 @@ describe("Recovery SDK", () => {
             }
         });
 
-        await acceptRequest(config, client, signer, alice);
-        await acceptRequest(config, client, signer, bob);
+        console.log("LO's - Accepting")
+        await acceptRequest(config, client, signer, alice, requester);
+        await acceptRequest(config, client, signer, bob, requester);
 
         const accepted = (await pending.refresh()) as AcceptedProtection;
-        let activate = await accepted.activate(signer);
+        console.log("Activating")
+        return accepted.activate(signer);
+    }
 
-        expect(activate).toBeInstanceOf(ActiveProtection);
-    });
+    it("recovers a lost account", async () => {
```

Comment:
Splitting the test into 2 cases requires the cases to be executed in the right order. Also, one generally does not expect a dependency between separate cases. Therefore, I would merge the 2 cases but keep your refactoring to make the "big" test case more readable.

Reply:
The 2 test cases are completely independent, and can be run in any order.
The test case `enable protection` runs using `REQUESTER_ADDRESS`.
The test case `recovers a lost account` runs using `LOST_ADDRESS` (and `NEW_ADDRESS`). For this latter case, a protection is enabled via the call `await enableProtection(LOST_ADDRESS)` at line 83.

Reply:
Yes.